src/lib/undux/utils/sidemenu.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import type { Store } from 'undux'
+import GDStore from '../GDStore'
+import { showDialogWithData, hideDialog } from './dialog'
+
+export const showSidemenu = (store: Store) => {
+  store.set('sidemenu')({ visible: true })
+}
+
+export const hideSidemenu = (store: Store) => {
+  store.set('sidemenu')({ visible: false })
+}
+
+export const toggleSidemenu = (store: Store) => {
+  store.set('sidemenu')({ visible: !store.get('sidemenu').visible })
+}
+
+export const useSidemenu = () => {
+  const store = GDStore.useStore()
+  return {
+    showDialogWithData: showDialogWithData.bind(null, store),
```

Comment:
showDialogWithData, and hideDialog should be there? Also are we using show, hide and toggle?

Reply:
Done. It shouldn´t be here